packages/liveblocks-client/src/live.ts

```diff
@@ -83,7 +85,7 @@ export enum ClientMessageType {
 
 export type ClientEventMessage = {
   type: ClientMessageType.ClientEvent;
-  event: any;
+  event: JsonObject;
```

Comment:
To be tested but I think it could be a `Json`

Reply:
This is fixed in 7b28f2542c7aaa2f68374cbcbce990b7dc31cf01. Also, I added an E2E test to test that broadcasting any `Json` value (not just objects) works. I couldn't find an existing test for the broadcast/event listener APIs, so I had to add a new one for it myself (see bd5dee884812a527adac5a95c0e4e46bd150b669).